public/css/efiling.css

```diff
@@ -22,7 +22,42 @@
   background-position: center;
   background-image: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='30px' height='30px' viewBox='0 0 100 100'%3E%3Ccircle fill='white' cx='50' cy='10' r='3'/%3E%3Ccircle fill='white' cx='78' cy='22' r='5'/%3E%3Ccircle fill='white' cx='90' cy='50' r='9'/%3E%3C/svg%3E")
 }
-.efiling-spinner::before {
+.efiling-spinner-blue {
+    width: 30px;
+    height: 30px;
+  animation: spin 1s linear infinite;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='30px' height='30px' viewBox='0 0 100 100'%3E%3Ccircle fill='rgb(0, 51, 102)' cx='50' cy='10' r='3'/%3E%3Ccircle fill='rgb(0, 51, 102)' cx='78' cy='22' r='5'/%3E%3Ccircle fill='rgb(0, 51, 102)' cx='90' cy='50' r='9'/%3E%3C/svg%3E")
```

Comment:
should we store this in a .svg to be consistent?  or move our other icons to code?